### PR TITLE
Update network environment manifest

### PR DESCRIPTION
**Pull request by yusufsheiqh**
## Context

- Add configured VPC to environment manifest file to support an additional Availability Zone (AZ)
  - The London (`eu-west-2`) region has 3 AZs, but AWS Copilot defaults to 2 AZs as some regions do not have 3 AZs.
  - Public subnets are required to be part of the configuration.
- Update `addons.parameters.yml` with the additional private subnet

## Additional Information

More information on why the build is failing - https://uktrade.atlassian.net/browse/DBTP-325

### CODE REVIEW

File: tests/bootstrap_strings.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+TEST_ENVIRONMENT_MANIFEST = "# The manifest for the \"test\" environment.\n# Read the full specification for the \"Environment\" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\nnetwork:\n  vpc:\n    cidr: '10.0.0.0/16'\n    subnets:\n      public:\n        - cidr: '10.0.0.0/24'\n          az: 'eu-west-2a'\n        - cidr: '10.0.1.0/24'\n          az: 'eu-west-2b'\n        - cidr: '10.0.2.0/24'\n          az: 'eu-west-2c'\n      private:\n        - cidr: '10.0.3.0/24'\n          az: 'eu-west-2a'\n        - cidr: '10.0.4.0/24'\n          az: 'eu-west-2b'\n        - cidr: '10.0.5.0/24'\n          az: 'eu-west-2c'\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n"
```
Comment (WillGibson) [inline tests/bootstrap_strings.py:1]: Maybe for a separate pull request, but could we do something like this to make these easier to grok and see what's actually changed?

```suggestion
TEST_ENVIRONMENT_MANIFEST = """# The manifest for the \"test\" environment.
# Read the full specification for the \"Environment\" type at:
#  https://aws.github.io/copilot-cli/docs/manifest/environment/

# Your environment name will be used in naming your resources like VPC, cluster, etc.
name: test
type: Environment

# Import your own VPC and subnets or configure how they should be created.
# network:
#   vpc:
#     id:
network:
  vpc:
    cidr: '10.0.0.0/16'
    subnets:
      public:
        - cidr: '10.0.0.0/24'
          az: 'eu-west-2a'
        - cidr: '10.0.1.0/24'
          az: 'eu-west-2b'
        - cidr: '10.0.2.0/24'
          az: 'eu-west-2c'
      private:
        - cidr: '10.0.3.0/24'
          az: 'eu-west-2a'
        - cidr: '10.0.4.0/24'
          az: 'eu-west-2b'
        - cidr: '10.0.5.0/24'
          az: 'eu-west-2c'

# Configure the load balancers in your environment, once created.

http:
  public:
    certificates:
      - ACM-ARN-FOR-test.landan.cloudapps.digital
#   private:


# Configure observability for your environment resources.
observability:
  container_insights: true
"""
```

**Review by WillGibson — APPROVED** [review]
Approving.

One suggestion that you may or may not want to pick up in this pull request or do separately.

File: tests/bootstrap_strings.py (hunk 1 above)
Comment (yusufsheiqh) [reply tests/bootstrap_strings.py:1]: Sure - I will do it as part of this PR, as I would like to test the multi-AZ functionality anyway.

**Review by lgarvey — APPROVED** [review]
Looks good!

**Comment by WillGibson** [comment]
Just trying the build, loks like they started failing yesterday for reason not related to the changed in this pull request, e.g. the ones for pull request #101 are failing with the same symptom.

**Comment by yusufsheiqh** [comment]
> Just trying the build, loks like they started failing yesterday for reason not related to the changed in this pull request, e.g. the ones for pull request #101 are failing with the same symptom.

@WillGibson Correct. I have added a JIRA ticket which is referenced in the description of this PR.

**Comment by WillGibson** [comment]
The Cloud Foundry pull request is merged and released https://github.com/cloudfoundry-community/cf-python-client/pull/195#issuecomment-1641782652

**Comment by yusufsheiqh** [comment]
The CodeBuild build project issue has now been resolved in https://github.com/uktrade/copilot-tools/pull/103.

File: tests/bootstrap_strings.py (hunk 1 above)
Comment (yusufsheiqh) [reply tests/bootstrap_strings.py]: Resolved in https://github.com/uktrade/copilot-tools/pull/102/commits/457fc8917c625c1a5d6a8e1bb89c401f077cd7a7.

Comment (WillGibson) [reply tests/bootstrap_strings.py]: Great approach. Love it.

**Review by WillGibson — APPROVED** [review]
Reapproving after you recent changes.

**Pull request merged**